xdocs/usermanual/functions.xml

```diff
@@ -1499,6 +1502,36 @@ becomes:
 </properties>
 </component>
 
+
+<component index="&sect-num;.5.22" name="__timeShifting">
+<description>
+    <p>The timeShifting function returns a date in various formats with the specified amount of seconds/minutes/hours/days/months added</p>
+    </description>
+<properties>
+        <property name="Format" required="No">
+        The format to be passed to DateTimeFormatter.
+        If omitted, the function use milliseconds since epoch format.
+        </property>
+        <property name="Date to shift" required="No">
+        Indicate the date with the format from first paramater to shift
+        If omitted, the date is set to now
+        </property>
+        <property name="value to shift" required="No">
+        Indicate the specified amount of seconds/minutes/hours/days/mounths to add according to this usage :
```

Comment:
mounths => months
seconds, minutes, ... or months

As a side note, in english microtypography there is no space in front of a colon, as is in french microtypography.